byway/src/main/webapp/recommendations.js

```diff
@@ -198,35 +165,53 @@ function findPlacesWithTextSearch(request, statuses) {
 
 /**
  * Reveals status codes if there was an issue with a request.
- * TODO: Make a class to indicate as a "warning" and make text red.
- * Also, organize on front end.
  * @param {Set} statuses String elements with status codes from placesService
  */
 function alertUser(statuses) {
-  document.getElementById("message-container").style.visibility = 'visible';
-  document.getElementById("general-message").style.visibility = 'visible';
-  let statusesContainer = document.getElementById("statuses");
-  statusesContainer.style.visibility = 'visible';
-  for(let status of statuses) {
-    let statusElement = document.createElement('ul');
-    statusElement.innerText = status;
-    statusesContainer.appendChild(statusElement);
+  let msgContainer = document.getElementById("message-container");
+  msgContainer.style.visibility = 'visible';
+  const allStatuses = formatStatusMessages(statuses);
+  msgContainer.innerText = "Showing limited results due to: " + allStatuses;
+}
+
+/**
+ * Go through status messages and add punctuation to make a sentence.
+ * @param {Set} statuses contains elements with status codes from placesService
+ * @return String with punctuation as it lists all status codes.
+ */
+function formatStatusMessages(statuses) {
+  if (statuses.size < 1) return "";
+  if (statuses.size == 1) {
+    for (let singleStatus of statuses) {
```

Comment:
Why do you need to loop through if there's only 1 status?

Reply:
Fixed

Reply:
Fixed